bundle.Dockerfile

```diff
@@ -23,4 +23,4 @@ COPY bundle/tests/scorecard /tests/scorecard/
 LABEL name="arkmq-org/activemq-artemis-operator-bundle"
 LABEL description="ActiveMQ Artemis Broker Operator Bundle"
 LABEL maintainer="Roddie Kieley <rkieley@redhat.com>"
```

Comment:
do we want to replace roddie's email with a reference to https://arkmq.org/

Reply:
Replacing it is a good idea, perhaps info@arkmq.org?

Reply:
or the back to the discussions?

Reply:
The email address info@arkmq.org for the maintainer makes sense to me because it allows to send confidential emails, we could add another annotation for the support that points to the GitHub discussions however if required.